apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -298,15 +299,22 @@ do_select(
 ) ->
     QueryState = maybe_apply_total_query(Node, QueryState0),
     Result =
-        case maps:get(continuation, QueryState, undefined) of
-            undefined ->
-                ets:select(Tab, Ms, Limit);
-            Continuation ->
-                %% XXX: Repair is necessary because we pass Continuation back
-                %% and forth through the nodes in the `do_cluster_query`
-                ets:select(ets:repair_continuation(Continuation, Ms))
+        try
+            case maps:get(continuation, QueryState, undefined) of
+                undefined ->
+                    ets:select(Tab, Ms, Limit);
+                Continuation ->
+                    %% XXX: Repair is necessary because we pass Continuation back
+                    %% and forth through the nodes in the `do_cluster_query`
+                    ets:select(ets:repair_continuation(Continuation, Ms))
+            end
+        catch
+            exit:_ = Exit -> {error, Exit};
+            Type:Exception -> {error, {Type, Exception}}
```

Comment:
If this rarely happens, (i.e. not going to flood the logs with the same stacktrace etc), it's ok to add all info. 
```suggestion
            Type:Reason:Stack -> {error, #{exception => Type, reason => Reason, stacktrace => Stack}
```

Reply:
Ack